src/lib.nr

```diff
@@ -24,3 +24,34 @@ mod tests;
 
 // Benchmarks
 mod benchmarks;
+
+// Re-export the pre-defined bignum types, for easier access:
+pub use fields::bls12_377Fq::BLS12_377_Fq;
+pub use fields::bls12_377Fr::BLS12_377_Fr;
+pub use fields::bls12_381Fq::BLS12_381_Fq;
+pub use fields::bls12_381Fr::BLS12_381_Fr;
+pub use fields::bn254Fq::BN254_Fq;
+pub use fields::ed25519Fq::ED25519_Fq;
+pub use fields::ed25519Fr::ED25519_Fr;
+pub use fields::mnt4_753Fq::MNT4_753_Fq;
+pub use fields::mnt4_753Fr::MNT4_753_Fr;
+pub use fields::mnt6_753Fq::MNT6_753_Fq;
+pub use fields::mnt6_753Fr::MNT6_753_Fr;
+pub use fields::pallasFq::Pallas_Fq;
+pub use fields::pallasFr::Pallas_Fr;
+pub use fields::secp256k1Fq::Secp256k1_Fq;
+pub use fields::secp256k1Fr::Secp256k1_Fr;
+pub use fields::secp256r1Fq::Secp256r1_Fq;
+pub use fields::secp256r1Fr::Secp256r1_Fr;
+pub use fields::secp384r1Fq::Secp384r1_Fq;
+pub use fields::secp384r1Fr::Secp384r1_Fr;
+pub use fields::U1024::U1024;
+pub use fields::U2048::U2048;
+pub use fields::U256::U256;
+pub use fields::U384::U384;
+pub use fields::U4096::U4096;
+pub use fields::U512::U512;
+pub use fields::U768::U768;
+pub use fields::U8192::U8192;
+pub use fields::vestaFq::Vesta_Fq;
+pub use fields::vestaFr::Vesta_Fr;
```

Comment:
I don't think that we should be exporting these from the root, maybe from `fields`.

Reply:
I went looking for a good "Fight me" gif. There was a pretty good one from the movie Troy. Remember that movie? But when I went to the site it seemed broken, so I can't use it. Please imagine a gif of the actor Brian Cox (not the physicist) shouting "Fight me!", dressed in ancient greek armour, holding a sword, standing in front of an army. I think he's shouting at Brad Pitt. You're Brad Pitt in this scenario. Please don't pass up on this opportunity to portray Brad Pitt. You know Brian Cox - he's in Succession - that HBO TV series. It's a pretty good show. Very good acting. Possibly too good, because all of the characters are despicable and unlikeable. One comes away from the show quite depressed, actually.

Reply:
:shrug:

Reply:
I WON THE FIGHT. It was the gif description that scared you off, right?